src/App.jsx

```diff
@@ -107,17 +107,34 @@ function App() {
                             key={e.id}
                             todo={e}
                         >
-                            <Button
-                                name={''}
-                                className={'button__delete'}
-                                onClick={() => onClickCrossButton(e?.id)}
-                            />
+                            <SubMenu >
+
+                                <SubMenuButton
+                                    onClick={null}
+                                    name={'done'}
+                                >
+                                </SubMenuButton>
+
+                                <SubMenuButton
+                                    onClick={null}
+                                    name={'edit'}
+                                >
+                                </SubMenuButton>
+
+                                <SubMenuButton
+                                    onClick={() => onClickCrossButton(e?.id)}
+                                    name={'delete'}
+                                >
+                                </SubMenuButton>
+
+                            </SubMenu>
                         </TodoCard>
                     ))}
                 </div>
             ) : (
                 <p>No Tasks Yet</p>
             )}
+
```

Comment:
Don't use ternary operator for displaying html. Good examples for ternaries:
```
const foo = condition ? 'foo' : 'bar';

return condition ? 'foo' : 'bar';

const VeeeeeeeeeeeeeeeryLoooooongCondiiiiiiiition
   ? callSomething()
   : callAnotherOneThing();
```

All other use cases make you code dirty. It should be used to make it clean and simple, not to make your life harder because you don't want to use `if/else` statement

Reply:
there is one more trick that make your code much readable in this cases.
You should avoid `if`, `elseif`, `else` situations. It's much better to let your code grow down instead of right.
// Bad
```
if (something) {
   doThing()
} else {
  doAnotherThing()
}
```

// Good
```
if (something) {
  doSomething();
}

doAnotherThing();
```

It's more clear to read and debug because of the way you read the code.
It also allows you to forget about some first logical block during debugging in case if `something` is negative condition